cmap error

a new error appeared this morniong in a CI run: 

```
geetools/FeatureCollection/__init__.py:278: in plot_by_features
    return self._plot(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

type = 'scatter'
data = ***'jan': ***'Desert': 27.95434081213815, 'Forest': 235.3735403009758, 'Grassland': 9.63[68](https://github.com/gee-community/geetools/actions/runs/9163784913/job/25193667712?pr=267#step:5:69)9298170773***, 'jun': ***'Desert': 1....13685962277756***, 'sep': ***'Desert': 37.412246641431544, 'Forest': 54.064533[71](https://github.com/gee-community/geetools/actions/runs/9163784913/job/25193667712?pr=267#step:5:72)5960614, 'Grassland': 45.833161420363155***
label_name = 'label', colors = [], ax = <Axes: >, kwargs = ***
props = ['Desert', 'Forest', 'Grassland']

    @staticmethod
    def _plot(
        type: str,
        data: dict,
        label_name: str,
        colors: list = [],
        ax: Optional[Axes] = None,
        **kwargs,
    ) -> Axes:
        """Plotting mechanism used in all the plotting functions.
    
        It binds the matplotlib capabilities with the data aggregated either by feature or by properties.
        the shape of the data should as follows:
    
        .. code-block::
    
            ***
                "label1": ***"properties1": value1, "properties2": value2, ...***
                "label2": ***"properties1": value1, "properties2": value2, ...***,
                ...
            ***
    
        Args:
            type: The type of plot to use. can be any type of plot from the python lib `matplotlib.pyplot`. If the one you need is missing open an issue!
            data: the data to use as inputs of the graph. please follow the fomrmat specified in the documentation.
            label_name: The name of the property that was used to generate the labels
            property_names: The list of names that was used to name the values. They will be used to order the keys of the data dictionary.
            colors: A list of colors to use for the plot. If not provided, the default colors from the matplotlib library will be used.
            ax: The matplotlib axes to use. If not provided, the plot will be send to a new figure.
            kwargs: Additional arguments from the ``pyplot`` chat type selected.
        """
        # define the ax if not provided by the user
        if ax is None:
            fig, ax = plt.subplots()
    
        # gather the data from parameters
        labels = list(data.keys())
        props = list(data[labels[0]].keys())
>       colors = colors if colors else plt.cm.get_cmap("tab10").colors
E       AttributeError: module 'matplotlib.cm' has no attribute 'get_cmap'

geetools/FeatureCollection/__init__.py:420: AttributeError
```

--- a/geetools/FeatureCollection/__init__.py
+++ b/geetools/FeatureCollection/__init__.py
@@ -363,7 +363,7 @@ class FeatureCollectionAccessor:
             raise ValueError("Pie chart can only be used with one property")
 
         kwargs["rwidth"] = kwargs.get("rwidth", 0.9)
-        kwargs["color"] = color or plt.cm.get_cmap("tab10").colors[0]
+        kwargs["color"] = color or plt.get_cmap("tab10").colors[0]
         ax.hist(list(data[labels[0]].values()), **kwargs)
         ax.set_xlabel(labels[0])
         ax.set_ylabel("frequency")
@@ -417,7 +417,7 @@ class FeatureCollectionAccessor:
         # gather the data from parameters
         labels = list(data.keys())
         props = list(data[labels[0]].keys())
-        colors = colors if colors else plt.cm.get_cmap("tab10").colors
+        colors = colors if colors else plt.get_cmap("tab10").colors
 
         # draw the chart based on the type
         if type == "plot":

